src/vs/workbench/services/soundNotification/browser/audioNotificationService.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IAudioNotificationService } from 'vs/editor/browser/services/audioNotificationService';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { registerSingleton } from 'vs/platform/instantiation/common/extensions';
+import { Registry } from 'vs/platform/registry/common/platform';
+import { Extensions as ConfigurationExtensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
+import { localize } from 'vs/nls';
+import { raceTimeout } from 'vs/base/common/async';
+
+export class AudioNotificationService implements IAudioNotificationService {
+	public readonly _serviceBrand: undefined;
+
+	constructor(@IConfigurationService private readonly _configurationService: IConfigurationService) {
+	}
+
+	private async playSound(fileName: string) {
+		if (!this._configurationService.getValue<boolean>('audioNotifications.enabled')) {
+			return;
+		}
+
+		const audio = new Audio(`../../../workbench/services/soundNotification/browser/media/${fileName}.webm`);
```

Comment:
Should we cache the played sounds? So we do not always load them? (just an idea)
I am also not sure if this is the proper way to load them.

Reply:
Let's introduce caching when necessary. Shouldn't the browser itself already cache the audio file on the network level?

Reply:
I do not know. But yes, we can add caching later if needed.